topo_processor/metadata/loaders/metadata_loader_tiff.py

```diff
@@ -14,8 +16,19 @@ class MetadataLoaderTiff(MetadataLoader):
     def is_applicable(self, item: Item) -> bool:
         return is_tiff(item.path)
 
-    def add_metadata(self, item: Item) -> None:
-        # TODO use GDAL to load metadata from the tiff file if it exists
+    async def add_metadata(self, item: Item) -> None:
+        with rasterio.open(item.path) as tiff:
+            # projection
+            if tiff.crs:
+                if "projection" not in item.stac_item.stac_extensions:
+                    item.stac_item.stac_extensions.append("projection")
+                item.stac_item.properties.update({"proj:epsg": epsg_code(tiff.profile["crs"])})
+            else:
+                item.stac_item.properties.update({"proj:epsg": None})
+            # tags
+            item.stac_item.properties.update({"linz:image_width": tiff.width, "linz:image_height": tiff.height})
```

Comment:
These should not be on the item but on the asset.

They should also not be a linz prefix? does STAC not have a image width/height option?

Reply:
The closest I can find is proj:shape under [here](https://github.com/radiantearth/stac-spec/blob/master/extensions/projection/README.md#item-properties-fields). Does it look like the one?